aiomisc/periodic.py

```diff
@@ -49,7 +52,9 @@ def periodic():
             del self._task
 
             self._task = self._loop.create_task(
-                (utils.shield(self._run) if shield else self._run)()
+                (utils.shield(self._run) if shield else self._run)(
+                    suppress_exceptions
+                )
```

Comment:
```python
run = utils.shield(self._run) if shield else self._run
self._task = self._loop.create_task(
    run(suppress_exceptions)
)
```
IMHO, this will better readable.